src/bundle-size.ts

```diff
@@ -6,10 +6,32 @@ type BuildManifest = {
   pages: Record<string, string[]>;
 };
 
+type ReactLoadableManifest = Record<string, { id: string, file: string }[]>;
+
 export type PageBundleSizes = { page: string; size: number }[];
 
-export function getBundleSizes(): PageBundleSizes {
-  const manifest = loadManifest();
+export function getStaticBundleSizes(): PageBundleSizes {
+  const manifest = loadBuildManifest();
+
+  const pageSizes: PageBundleSizes = Object.entries(manifest.pages).map(([page, files]) => {
+    const size = files
+      .map((filename: string) => {
+        const fn = path.join(process.cwd(), '.next', filename);
+        const bytes = fs.readFileSync(fn);
+        const gzipped = zlib.gzipSync(bytes);
+        return gzipped.byteLength;
+      })
+      .reduce((s: number, b: number) => s + b, 0);
+
+    return { page, size };
+  });
+
+  return pageSizes;
+}
```

Comment:
This is the same code as below? `getBundleSizes(manifest)` would remove the duplication?

Reply:
Extracted out the page size calculation. Does that make sense?